src/universal/components/MeetingContentHeader.tsx

```diff
@@ -12,26 +15,61 @@ const MeetingContentHeaderStyles = styled('div')({
   justifyContent: 'space-between',
   margin: 0,
   maxWidth: '100%',
-  padding: '0 1rem 1rem',
+  padding: '12px 16px 32px',
   width: '100%',
-  [minWidthMediaQueries[3]]: {
-    padding: '0 1rem 2rem'
+  [meetingTopBarMediaQuery]: {
+    padding: '16px 16px 32px'
   }
 })
 
 const HeadingBlock = styled('div')({
-  alignItems: 'flex-start',
+  alignItems: 'center',
   display: 'flex',
-  flex: 1,
-  padding: '1.25rem 0 1rem'
+  minHeight: 32,
+  [localHeaderBreakpoint]: {
+    flex: 1
+  },
+  [meetingTopBarMediaQuery]: {
+    alignItems: 'flex-start'
+  }
 })
 
+const PrimaryActionBlock = styled('div')({
+  order: 2,
+  [localHeaderBreakpoint]: {
```

Comment:
why create a new row? the hamburger is less than the width of 1 avatar, so we're not saving any space
![Screenshot from 2019-06-11 07-14-14](https://user-images.githubusercontent.com/5514175/59279300-8f223c00-8c18-11e9-92c8-d34f33f8ced8.png)

if it is the demo, we know only 3 avatars will be present, so we'll have enough space to fit them.

Reply:
I left comments in Figma, basically it shouldn’t wrap like if there is room. I’ll double check.